beep/tests/test_arbin_schedule.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 Toyota Research Institute. All rights reserved.
+"""Unit tests related to Generating Arbin Schedule files"""
+
+import os
+import unittest
+import difflib
+import json
+from beep import SCHEDULE_TEMPLATE_DIR
+from beep.protocol_tools.arbin_schedule_file import ScheduleFile
+
+TEST_DIR = os.path.dirname(__file__)
+TEST_FILE_DIR = os.path.join(TEST_DIR, "test_files")
+
+
+class ArbinScheduleTest(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_dict_to_file(self):
+        sdu = ScheduleFile(version='0.1')
```

Comment:
really minor suggestion, in case you want a cleaner file handling, you can use ScratchDir from monty, e.g. checkout https://github.com/TRI-AMDD/CAMD/blob/master/camd/tests/test_loop.py

Reply:
Yeah, I was considering that, but I kept needing to inspect the file. Should probably change that over as soon as we get the code in a good place